lib/scsi_defs.h

```diff
@@ -1,10 +1,13 @@
 #ifndef LIB_SCSI_DEFS_H
 #define LIB_SCSI_DEFS_H
 
-#include <cstdint>
+#include "absl/base/attributes.h"
+#include "absl/base/casts.h"
```

Comment:
Is this include used other than for uint_t's? #include <cstdint> might be a more concise include if that's the case.